### PR TITLE
Simplified fix whitespace

### PR DESCRIPTION
**Pull request by maxtaco**
For the 15 Sep fix, with some comments by @oconnor663 folded in.

### CODE REVIEW

File: src/sigchain/sigchain.iced [hunk 1]
```diff
@@ -8,7 +8,10 @@ ie = require('iced-error')
 UID_LEN = 32
 exports.SIG_ID_SUFFIX = SIG_ID_SUFFIX = "0f"
 
-
+strip_final_newline = (buf) ->
+  s = buf.toString('utf8')
```
Comment (oconnor663) [inline src/sigchain/sigchain.iced]: If you want to you can check `buf[-1...].equals Buffer "\n"` without doing the string conversion. I don't have a strong preference either way.

**Comment by oconnor663** [comment]
Looks good. Do we have a test for this, or just testing by hand?

**Comment by maxtaco** [comment]
Was testing it by hand.  Any ideas on how to test?

**Comment by oconnor663** [comment]
The plan: add a new test vector by copying pnelson's sigchain up to link 21.

**Comment by oconnor663** [comment]
I landed your commit in keybase-test-vectors just now.

**Pull request merged**